src/jobs/models/jobManager.js

```diff
@@ -8,7 +8,8 @@ const logger = require('../../common/logger'),
     databaseConnector = require('./database/databaseConnector');
 
 let cronJobs = {};
-const JOB_PLATFORM_NAME = 'predator.%s';
+const PREDATOR_RUNNER_PREFIX = 'predator';
```

Comment:
this resembles more the `PREDATOR_RUNNER_JOB_APP_LABEL`

Reply:
true but only when kubernetes is relevant